lib/wallet/integration/src/Test/Integration/Faucet.hs

```diff
@@ -2418,6 +2418,7 @@ shelleyIntegrationTestFunds = mconcat
     preregKeyWalletFunds = map ((,defaultAmt) . Address . unsafeFromHex)
         [ "6199a7c32aaa55a628d936b539f01d5415318dec8bcb5e59ec71af695b"
         , "61386c7a86d8844f4085a50241556043c9842d72c315c897a42a8a0510"
+        , "614adf026163a9182cd4087acc966aef46e37474a2994a06a69de44d6f"
```

Comment:
If we rely on having enough spare UTxOs for failing pending txs not to consume them all, I think we should add more.

Actually, we might as well not hard-code the addresses, but use the same approach as for the `seqMnemonics`. We could

1. Define `preregStakeKeyWallet :: Mnemonic 15`
2. Import it from the test itself (optional, but why not) https://github.com/input-output-hk/cardano-wallet/blob/56784ab870f827d223c3dd129d3eb5cc23cad3a5/lib/wallet/integration/src/Test/Integration/Scenario/API/Shelley/Migrations.hs#L593
3. Generate 10 addresses automatically like this https://github.com/input-output-hk/cardano-wallet/blob/56784ab870f827d223c3dd129d3eb5cc23cad3a5/lib/wallet/integration/src/Test/Integration/Faucet.hs#L2366, but with the `preregStakeKeyWallet` mnemonic.

Reply:
Done